resources/eventing/charts/controller/values.yaml

```
@@ -0,0 +1,18 @@
+# override name to avoid collision with kantive eventing resources
+nameOverride: subscription-controller
+replicaCount: 1
+serviceAccount:
+  # name defines optionally another name than the default name for the service account
+  name: ""
+
+clusterRole:
+  # name defines optionally another name than the default name for the cluster role
+  name: ""
+
+image:
+  # name is the name of the container image for the event-publisher-proxy
+  name: "eventing-controller"
+  # tag is the container tag of the event-publisher-proxy image
+  tag: "PR-9750"
```

Let's use the latest available image from this PR: https://github.com/kyma-project/kyma/pull/9744
```suggestion
  tag: "PR-9744"
```

---

Not applicable as we have `PR-9868`